image-swipe.d.ts

```diff
@@ -17,6 +17,11 @@ import { EventData } from "data/observable";
 import { CoercibleProperty, Property } from "ui/core/view";
 import { ItemsSource } from "ui/list-picker";
 import { ScrollView } from "ui/scroll-view";
+import { Cache } from "ui/image-cache";
+
+export class ImageSwipeBase {
```

Comment:
Exporting the base class is a bad thing. Since we are extending the base class the cache should be visible on the main `ImageSwipe` class. But seems there is a bug in {N} that does not copy static objects from the base class. So I suggest for now to remove this and have it available as part of the main class later once the bug is fixed. Or for now we can add the property in the platform specific files and change its name to not be prefixed with `_` since now it will be available for outside usage.